x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterQueriesWithInvalidLicenseIT.java

```diff
@@ -58,6 +58,7 @@ public void testMetadataCrossClusterQuery() {
         assertThat(e.getMessage(), containsString(LICENSE_ERROR_MESSAGE));
     }
 
+    @AwaitsFix(bugUrl = "es-12487")
```

Comment:
It is an interesting issue.
This test verifies index resolution with something like `FROM fake-remote:events`.

Field caps relies on `returnLocalAll = true` at https://github.com/elastic/elasticsearch/blob/18286be8df9a8e540dba1eb76bcfb0ac84a67678/server/src/main/java/org/elasticsearch/action/fieldcaps/TransportFieldCapabilitiesAction.java#L171-L172
that causes it to return all indices from local clusters when all remotes are not found.

There are existing issues about it: #115262 and #115872

Today this flag is not exposed via `FieldCapabilitiesRequest` so I am not sure how this could be fixed.

Reply:
Depends on https://github.com/elastic/elasticsearch/pull/134284